From: Pantelis Koukousoulas <pktoss@gmail.com>
Date: Sun, 9 Sep 2012 11:54:09 +0300
Subject: Disable ALSA by default

Based on a patch from Gergely Nagy who writes:

> The configure script is broken in many, many ways, so it's best to
> just disable ESD and ALSA in the beginning..

but reduced to just disabling ALSA, since we want to use the ESD protocol
to talk to PulseAudio's ESD emulation.

[commit message added -smcv]
Reviewed-by: Simon McVittie <smcv@debian.org>
---
 configure.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 3f8b6b6..2a6a86c 100644
--- a/configure.in
+++ b/configure.in
@@ -22,7 +22,7 @@ AM_MAINTAINER_MODE
 
 libmikmod_driver_af=yes
 libmikmod_driver_aix=no
-libmikmod_driver_alsa=yes
+libmikmod_driver_alsa=no
 libmikmod_driver_esd=yes
 libmikmod_driver_hp=no
 libmikmod_driver_oss=yes
